nomad/repo.py

```diff
@@ -94,18 +98,15 @@ def available(self):
 
     @cachedproperty
     def appliednames(self):
-       return [x for (x, ) in
-               self.engine.query('SELECT name FROM %s ORDER BY date' %
-                                 self.conf['nomad']['table'])]
+        q = 'SELECT name FROM {0} ORDER BY date'.format(
```

Comment:
I would prefer seeing `%s` here. They are not going anywhere, take less space that `{0}`, and I just don't see the point.